modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataLayoutResourceImpl.java

```diff
@@ -239,8 +241,12 @@ public Response postDataLayoutContext(
 			contextHttpServletRequest);
 		ddmFormRenderingContext.setHttpServletResponse(
 			contextHttpServletResponse);
+
 		ddmFormRenderingContext.setLocale(
-			contextAcceptLanguage.getPreferredLocale());
+			LocaleUtil.fromLanguageId(
+				contextHttpServletRequest.getHeader(
+					HttpHeaders.ACCEPT_LANGUAGE)));
+
```

Comment:
Did the SF add these empty lines before and after `ddmFormRenderingContext.setLocale(...) `?
If not, I think we should remove it.